components/appointmentPicker/appointmentPickerController.js

```diff
@@ -29,7 +29,9 @@ define('appointmentPickerController', [], function() {
 								// called for every date before it is displayed
 								beforeShowDay: (date) => {
 									if ($scope.hasDate(date)) {
-										if ($scope.hasTimeSlotsRemaining(date)) {
+										if (isDateTomorrowOrBefore(date)) {
```

Comment:
What happens if a signed in volunteer tries to schedule an appointment day of / tomorrow. Does this make it so they can't? In the previous world, they would've been able to.

There's some code in `getAppointments.php` that enables signed-in volunteers to schedule even for the previous hour. I'm thinking that maybe a better approach here would be to limit the availability in the PHP backend rather than doing this `isDateTomorrowOrBefore` in the FE. But I'm slightly torn

Reply:
Very good call. Made the change.